selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/opendeclaration/Eclipse0120Test.java

```diff
@@ -29,9 +29,9 @@
 
 /** @author Aleksandr Shmaraev */
 public class Eclipse0120Test {
-  private static final String PATH_TO_PACKAGE_PREFIX = "/src/main/java/org/eclipse/qa/examples/";
-  private static final String PROJECT_NAME =
-      NameGenerator.generate(Eclipse0120Test.class.getSimpleName(), 4);
+  private static final String PROJECT_NAME = NameGenerator.generate("Eclipse0120Test-", 4);
+  private static final String PATH_TO_EXPAND =
+      String.format("%s/%s", PROJECT_NAME, "src/main/java/org.eclipse.qa.examples");
```

Comment:
In my opinion, concatenation is better for this case.

Reply:
Why do you think so?

Reply:
Cleaner and more laconic entry.

Reply:
@dmytro-ndp 
As I see, you agree with the opinion of the @Ohrimenko1988.
Couldn't  you explain, why the concatenation is better in this case?

Reply:
I like you use **String.format()** to make code more readable. And it works well in case of more than one substitutions at the middle of line, or long strings, or the complex format of substitute.
But in this case, where we have one substitution at the start of line, concatenation looks really clearer, IMHO.

Reply:
Thank you for your explanation